Cargo.toml

```diff
@@ -11,6 +11,7 @@ bincode = "1.3.3"
 bytes = "1.1.0"
 cap = "0.1.0"
 clap = { version = "3.0.0", features = ["derive"] }
+click = "0.5.4"
```

Comment:
`Cargo.toml` is only for Rust dependencies. https://crates.io/crates/click isn't what you expected here. :)

The Python scripts in this repo don't use any environment manager like conda or poetry. We could just ask people to run the script inside an environment with click, or we could prescribe conda or poetry, or we could tell people to `pip install click`. Unless we wind up with lots of python scripts in this repo (unlikely), I'm not sure what to do. What's typical in a project you've encountered that has lots of random scripts, some of which have a few dependencies?

Reply:
- [X] add .DS_Store to .gitignore, I guess I should remove this PR and create a new one for your review?
- [ ]  I dont really know, I dont see the issue of having click here. My constrain here is I dont think people should copy and paste the shell outcome to create a new csv, the tool should make that easier for them. We should provide an easy way to get the new study areas, so I was attempting that.